internal/runtime/safecall_test.go

```diff
@@ -70,4 +71,28 @@ func TestSafeCall(t *testing.T) {
 	out, err = SafeCall(fn10, nil, nil, nil)
 	require.NoError(t, err)
 	assert.Equal(t, "crap", out)
+
+	// Test to send a function with more than 2 returns
+	fn13 := func() (string, string, error) { return "a", "b", errors.New("c") }
+	out, err = SafeCall(fn13)
+	require.ErrorContains(t, err, "c")
+	assert.Equal(t, "a", out)
```

Comment:
```suggestion
	fn13 := func() (string, string, error) { return "a", "b", errors.New("whatever") }
	out, err = SafeCall(fn13)
	require.ErrorContains(t, err, "whatever")
	assert.Equal(t, "a", out)
```

c is so common that it could be added in a wrapper with a refactoring, but the would be broken

Also, in fact the code should be this

```suggestion
	testErr := errors.New("c") 
	fn13 := func() (string, string, error) { return "a", "b", testErr}
	out, err = SafeCall(fn13)
	require.ErrorIs(t, err, testErr)
	assert.Equal(t, "a", out)
```